src/www/views/servers_view/index.ts

```diff
@@ -108,8 +108,14 @@ Polymer({
         </div>
         <div
           class="footer subtle"
+          hidden$="[[showAltAccessMessage]]"
```

Comment:
Ditto. Also for the changes below.

Reply:
Done

Reply:
I've also replaced two unrelated uses of `hidden` here with dom-if.